microsoft-edge/devtools-guide-chromium/ie-mode/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Use DevTools in Internet Explorer mode
+title: Use DevTools in IE mode
```

Comment:
Looking at https://www.microsoft.com/en-us/edge/business/ie-mode it seems we call use the longer `Internet Explorer mode` form instead of `IE mode`.

Reply:
outdated.  expanded.